cmd/soci/commands/create.go

```diff
@@ -95,16 +90,15 @@ var CreateCommand = cli.Command{
 			return err
 		}
 
-		manifestType := soci.ManifestOCIArtifact
-		if cliContext.Bool(createORASManifestFlag) {
-			manifestType = soci.ManifestORAS
+		ib, err := soci.NewIndexBuilder(cs,
+			soci.WithSpanSizeOption(spanSize),
+			soci.WithMinLayerSizeOption(minLayerSize),
+			soci.WithBuildToolIdentifierOption(buildToolIdentifier))
```

Comment:
Do they need Option suffix?

Reply:
This is according to Dave's @dvnguyen-amzn  [refactored index builder code](https://github.com/awslabs/soci-snapshotter/pull/158/commits/938d6fb06e3fa180a84ce0102d120ccd3318cd9d#diff-d3c6854bbb391053f98290fc7eff03bffa09b17b72101f8babe6fcd5744983daR65-R84).

Reply:
Since you're removing similar functions in `soci_index.go` in this PR, I think you can also remove the `Option` suffix here by renaming them.

Reply:
I'm fine keep them as is, but I'd like to rename them later.